2021-05-18 02:21:20,623 INFO    MainThread:17642 [wandb_setup.py:_flush():69] setting env: {}
2021-05-18 02:21:20,623 INFO    MainThread:17642 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-18 02:21:20,623 INFO    MainThread:17642 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/code/mmdetection_trash/wandb/run-20210518_022120-21xdxrec/logs/debug.log
2021-05-18 02:21:20,623 INFO    MainThread:17642 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/code/mmdetection_trash/wandb/run-20210518_022120-21xdxrec/logs/debug-internal.log
2021-05-18 02:21:20,623 INFO    MainThread:17642 [wandb_init.py:init():369] calling init triggers
2021-05-18 02:21:20,624 INFO    MainThread:17642 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {'epoch': 60, 'batch_size': 32, 'optimizer': {'type': 'AdamW', 'lr': 0.0001, 'weight_decay': 0.0001}, 'optimizer_config': {'grad_clip': {'max_norm': 35, 'norm_type': 2}, 'type': 'OptimizerHook'}, 'lr_config': {'warmup': 'linear', 'warmup_iters': 200, 'warmup_ratio': 0.1, 'periods': [20, 12, 12, 12, 12], 'restart_weights': [1, 0.7, 0.6, 0.5, 0.4], 'min_lr_ratio': 2e-06, 'type': 'CosineRestartLrUpdaterHook'}}
2021-05-18 02:21:20,624 INFO    MainThread:17642 [wandb_init.py:init():391] re-initializing run, found existing run on stack: 1yxp3x3k
2021-05-18 02:21:21,298 INFO    MainThread:17642 [wandb_run.py:finish():1208] finishing run openingsound/stage_3_object_det/1yxp3x3k
2021-05-18 02:21:21,336 INFO    MainThread:17642 [jupyter.py:save_history():320] not saving jupyter history
2021-05-18 02:21:21,336 INFO    MainThread:17642 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-18 02:21:21,336 INFO    MainThread:17642 [wandb_init.py:_jupyter_teardown():268] cleaning up jupyter logic
2021-05-18 02:21:21,336 INFO    MainThread:17642 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 0
2021-05-18 02:21:21,337 INFO    MainThread:17642 [wandb_run.py:_restore():1507] restore
2021-05-18 02:21:21,342 INFO    MainThread:17642 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1052
  total_bytes: 1052
}

2021-05-18 02:21:21,606 INFO    MainThread:17642 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1052
  total_bytes: 1052
}

2021-05-18 02:21:22,232 INFO    MainThread:17642 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 1052
  total_bytes: 6969
}

2021-05-18 02:21:23,037 INFO    MainThread:17642 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11061
  total_bytes: 11061
}

2021-05-18 02:21:23,139 INFO    MainThread:17642 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11061
  total_bytes: 11061
}

2021-05-18 02:21:24,227 INFO    MainThread:17642 [wandb_run.py:_show_files():1879] logging synced files
2021-05-18 02:21:24,232 INFO    MainThread:17642 [wandb_init.py:init():418] starting backend
2021-05-18 02:21:24,232 INFO    MainThread:17642 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-18 02:21:24,235 INFO    MainThread:17642 [backend.py:ensure_launched():132] starting backend process...
2021-05-18 02:21:24,296 INFO    MainThread:17642 [backend.py:ensure_launched():137] started backend process with pid: 18705
2021-05-18 02:21:24,298 INFO    MainThread:17642 [wandb_init.py:init():423] backend started and connected
2021-05-18 02:21:24,316 INFO    MainThread:17642 [wandb_init.py:init():465] updated telemetry
2021-05-18 02:21:24,317 INFO    MainThread:17642 [wandb_init.py:init():484] communicating current version
2021-05-18 02:21:25,097 INFO    MainThread:17642 [wandb_init.py:init():489] got version response 
2021-05-18 02:21:25,097 INFO    MainThread:17642 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-18 02:21:25,328 INFO    MainThread:17642 [wandb_init.py:init():522] starting run threads in backend
2021-05-18 02:21:29,946 INFO    MainThread:17642 [wandb_run.py:_console_start():1565] atexit reg
2021-05-18 02:21:29,946 INFO    MainThread:17642 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.WRAP
2021-05-18 02:21:29,947 INFO    MainThread:17642 [wandb_run.py:_redirect():1455] Wrapping output streams.
2021-05-18 02:21:29,947 INFO    MainThread:17642 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-18 02:21:29,947 INFO    MainThread:17642 [wandb_init.py:init():546] run started, returning control to user process
